### PR TITLE
Added method to update the profile name

### PR DESCRIPTION
**Pull request by edgardmessias**
This depends of #277 to work from phone update

To test (it takes a while): `npm install github:edgardmessias/Baileys#feat_pushname`

### CODE REVIEW

**Comment by adiwajshing** [comment]
Looks good. It might be worth it to add a `contact-update` event which takes in the name updates -- even for the `Conn,pushname` event as well

**Comment by edgardmessias** [comment]
What you mean?

**Comment by adiwajshing** [comment]
> What you mean?

Just like there is `chat-update` -- could have an event for `contact-update`?

**Comment by edgardmessias** [comment]
Now I understand, I will update it to do that.

**Comment by edgardmessias** [comment]
@adiwajshing done

File: src/WAConnection/4.Events.ts [hunk 1]
```diff
@@ -298,9 +298,18 @@ export class WAConnection extends Base {
             }
         })
         // status updates
-        this.on('CB:Status', async json => {
+        this.on('CB:Status,status', async json => {
             const jid = whatsappID(json[1].id)
             this.emit ('user-status-update', { jid, status: json[1].status })
```
Comment (adiwajshing) [inline src/WAConnection/4.Events.ts:303]: Should probably deprecate `user-presence-update`

Comment (edgardmessias) [reply src/WAConnection/4.Events.ts]: done

**Comment by edgardmessias** [comment]
Ping @adiwajshing

**Comment by adiwajshing** [comment]
Looks good, merging.

**Pull request merged**